translate/gast_to_js.py

```diff
@@ -0,0 +1,65 @@
+
+
+def list_to_csv_str(l):
+    s = ""
+    for i in l:
+        s += str(i) + ", "
+    
+    return s[:-2] # remove last comma and space
+
+
+def logStatement(gast):
+    arg_string = list_to_csv_str(gast["args"])
+    return "console.log(" + arg_string + ")"
+
+
+def value_to_str(val):
+    if type(val) == str:
+        return '"' + val + '"'
+    else:
+        return str(val)
+
+
+def varAssign(gast):
+    value = value_to_str(gast["varValue"])
+    return "const " + gast["varId"] + " = " + value
```

Comment:
Not sure const is the right one here, variables may be reassigned lower down in the code.

Reply:
This is a place-holder for now- i will add a TODO